pylele.sh

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env bash
 
-time python3 ./pylele.py $@ > pylele.log
\ No newline at end of file
+rm -rf ./build
```

Comment:
this is probably too much and unexpected.  the latest default behavior is every export is a subdir under build/ with datetime stamp allowing for multiple exports to co-exist. this will clean all the history and may be a surprise to the user.

Reply:
sure you are right: i noticed that only after submitting the pull request. The new behavior with  new folder for each build is perfect.